doc/api/http.md

```diff
@@ -901,6 +901,10 @@ also be accessed at `request.connection`.
 This event can also be explicitly emitted by users to inject connections
 into the HTTP server. In that case, any [`Duplex`][] stream can be passed.
 
+If `socket.setTimeout()` is called here, note that the timeout will
```

Comment:
```suggestion
If `socket.setTimeout()` is called here, the timeout will
```

Reply:
@Trott Are we landing as is or can we add this fix on landing?

Reply:
@inversion are you ok with @Trott suggestion? If so please accept it so this can land.

Reply:
My change is non-blocking. It's not crucial that `note that` be removed. (It can also be removed by someone else who is landing this--just push to the branch. Or it can be removed in a subsequent PR as I imagine there are lots of opportunities for cleanup in the doc file.)